### PR TITLE
Added scheduled runs of SCA workflow

### PR DESCRIPTION
**Pull request by pete-j-g**

### CODE REVIEW

File: .github/workflows/terraform-static-analysis.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 name: Terraform Static Code Analysis
 
 on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '* 7 * * 1-5'
```
Comment (adeweetman-al) [inline .github/workflows/terraform-static-analysis.yml:6]: I read this as every minute past 7am, Mon-Fri. Should this not be: 0 7 * * 1-5

Comment (pete-j-g) [reply .github/workflows/terraform-static-analysis.yml:6]: 🤦 you are 100% correct, I'll get it updated in a few

Comment (pete-j-g) [reply .github/workflows/terraform-static-analysis.yml]: cron expression updated

**Review by adeweetman-al — APPROVED** [review]
Looks good.

**Pull request merged**